packages/libnvidia-container/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "libnvidia-container"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+
+[lib]
+path = "pkg.rs"
+
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/NVIDIA/libnvidia-container/archive/v1.5.0.tar.gz"
+sha512 = "da33212a7b169a21d7db8821a09ab351604a678d857217ca9de237cf0d0ae4e5a11146fae773276d42c06e1b1a056889275c1a72588004e2cab5528c91c5c1a9"
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/NVIDIA/nvidia-modprobe/archive/450.57/nvidia-modprobe-450.57.tar.gz"
```

Comment:
For lack of a better place to ask - should we be using `nvidia-modprobe` to load the nvidia drivers rather than just plain `modprobe`?